test/e2e/common/configmap_volume.go

```diff
@@ -57,9 +57,9 @@ var _ = Describe("[sig-storage] ConfigMap", func() {
 	})
 
 	/*
-		    Testname: configmap-nomap-user
-		    Description: Make sure config map without mappings works by mounting it
-			to a volume with a custom path (mapping) on the pod as non-root.
+		Release : v1.9
+		Testname: ConfigMap without mapping, non-root user
+		Description: Create a configMap, create a Pod that mounts a volume and populates the volume with data stored in the configMap. The config map that is created MUST be accessible to read from the newly created Pod using the volume mount. The Pod MUST be run as a specific non root user with uid 1000, and the data content of the file MUST be readable and file modes MUST be set to default 0x644.
```

Comment:
ordering/wording confuses me here, I think "Pod MUST be run as a specific non root user..." is a precondition?

ie:

Description: "A Pod mounts a ConfigMap as a volume.  The Pod is run as a non-root user with uid 1000.  The contents of the ConfigMap MUST be readable within the Pod as files on the volume, with their modes set 0x644"

Reply:
Fixed.